### PR TITLE
fixes #38 #39:Copy LBTCL and component restructure

### LINKED ISSUE #38: [PB] Copy LBTCL with PB Cohort details
Copy LBTCL with PB Cohort details

### PR DESCRIPTION
**Pull request by keraliss**

### CODE REVIEW

**Comment by emjshrx** [comment]
Looks good but add place holder text to other pages as well so that its easy for Raj and Setu to edit the content

**Comment by emjshrx** [comment]
Or you can spice it up with https://www.lipsum.com/

**Comment by keraliss** [comment]
Already changed, need me to push another commit?

File: src/pages/bpd.astro [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+            <p class="w-4/6 pl-5 text-xl font-bold lg:w-10/12">
+              {item.name}
+            </p>
+            <p class="w-2/6 lg:w-1/12">{item.chNo}</p>
+          </div>
+          <hr class=" border-spacing-12 border-dashed" />
+        </>
+      ))
+    }
+  </div>
```
Comment (emjshrx) [inline src/pages/bpd.astro:223]: This seems to be the same styling for all pages, can we move it into syllabus component?

File: src/pages/bpd.astro [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+
+    <ul slot="takeaways" class="list-disc">
+      {
+        takeaways.map((takeaway) => (
+          <li class="my-4 lg:text-xl">
+            <span class="font-bold">{takeaway.name}</span> :{" "}
+            {takeaway.desc}
+          </li>
+        ))
+      }
```
Comment (emjshrx) [inline src/pages/bpd.astro:143]: I think this too can be in takeaways component

File: src/pages/bpd.astro [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+        <li>
+          Become comfortable with Unix/Linux command-line
+          terminals
+        </li>
+      </ol>
+      <div
+        class="my-10 grid grid-cols-2 gap-10 lg:grid-cols-4"
+      >
+        {
+          format.map((item) => (
```
Comment (emjshrx) [inline src/pages/bpd.astro:163]: Same here. We can just pass the array as prop to the respective component

**Comment by emjshrx** [comment]
> Already changed, need me to push another commit?

No no thats fine

**Review by emjshrx — APPROVED** [review]

**Pull request merged**